README.md

```diff
@@ -8,10 +8,11 @@ This code base is meant to categorize each line of a *python* script several dif
 * conditionals
 * function defs
 * assignments
-
-# To be supported
 * function calls
 * hash lines and storage
+
+# To be supported
+* refactor, so that the multiline and other passed in variables are less important/all over the place also refactor to get the hash tool not to be so not readability for what changed..
```

Comment:
Two todos, one is a refactor to test the hash refactoring usefullness to catch errors, the other is to make the hash tool more readable.